ui/package.json

```diff
@@ -3,16 +3,16 @@
   "version": "0.0.1",
   "description": "Firefox System Build info manager.",
   "scripts": {
-    "build": "elm-make Main.elm --output=build/app.js && npm run copy-assets",
-    "copy-assets": "copyfiles index.html style.css build/",
-    "debug": "elm-live Main.elm --output=app.js --debug",
+    "build": "npm run copy-assets && elm-make src/Main.elm --output=build/app.js",
+    "copy-assets": "rimraf build && mkdir build && cp -r public/* build",
```

Comment:
TIL about `rimraf`. Do you want to add it to the npm dependencies so we're sure it's available on all platforms? (Eg osx, windows...?)